src/js/mainSectionImage.js

```diff
@@ -18,29 +23,64 @@ const resetCLassIcon = () => {
   cardDecoration.classList.remove('decoR', 'decoB', 'decoN');
 };
 
-const resetRadioButtons = () => {
-  for (const button of formRadioButtons) {
-    button.checked = false;
-  }
+const resetTextCard = () => {
+  defaultNameCard.innerHTML = 'Nombre Apellido';
+  defaultJobCard.innerHTML = 'Front-end developer';
 };
 
 const resetProfileImage = () => {
   uploadProfileImage.style.backgroundImage = '';
 };
 
-const resetTextCard = () => {
+// Resetear formulario
+
+const resetRadioButtons = () => {
+  for (const button of formRadioButtons) {

```

Comment:
Aunque el funcionamiento sea el mismo igual es buena idea homogeneizar todos los bucles con la misma sintaxis , la mayoria son for Each